pyathena/calc_cursor.py

```diff
@@ -0,0 +1,200 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from __future__ import annotations
+
+import logging
+import time
+from typing import Any, Dict, List, Optional, Union, cast
+
+from pyathena import NotSupportedError, OperationalError, ProgrammingError
+from pyathena.common import BaseCursor
+from pyathena.model import (
+    AthenaCalculationExecution,
+    AthenaQueryExecution,
+    AthenaSession,
+)
+from pyathena.util import retry_api_call
+
+_logger = logging.getLogger(__name__)  # type: ignore
+
+
+class CalcCursor(BaseCursor):
```

Comment:
suggestion (non-blocking): I think it might be better to name it as SparkCursor to be more explicit but I will leave that up to you

Reply:
Add spark package & Rename to SparkCursor: https://github.com/laughingman7743/PyAthena/pull/497/commits/f286cfe56f2c6c49f081fa2c63c1dcfaa7bacde9

Reply:
Add spark package & Rename to SparkCursor: https://github.com/laughingman7743/PyAthena/pull/497/commits/f3d49beab4eecf90bda5017e1342e576ce44b6c4